source/installation-guide/compatibility_matrix/index.rst

```diff
@@ -20,7 +20,7 @@ Below it shown the compatibility table for the Wazuh manager. It is recommended
 +----------------------------+--------------------------------------------------------------------------------+
 |                            |   **Wazuh version**                                                            |
 +    **Operating System**    +--------+--------+--------+--------+--------+--------+--------+--------+--------+
-|                            |  2.1.0 |  2.1.1 |  3.0.0 |  3.1.0 |  3.2.0 |  3.2.1 |  3.2.2 |  3.2.3 |  3.2.4 |
+|                            |  2.1.0 |  2.1.1 |  3.0.0 |  3.1.0 |  3.2.0 |  3.2.1 |  3.2.2 |  |WAZUH_LATEST| |  |WAZUH_LATEST| |
```

Comment:
Why? Do we need the same value in both?

Reply:
No, that's an error by myself, I'll fix it.